core/server/proxy/src/main/java/alluxio/proxy/PathsRestServiceHandler.java

```diff
@@ -107,18 +107,15 @@ public PathsRestServiceHandler(@Context ServletContext context) {
   @Consumes(MediaType.APPLICATION_JSON)
   public Response createDirectory(@PathParam("path") final String path,
       final CreateDirectoryPOptions options) {
-    return RestUtils.call(new RestUtils.RestCallable<Void>() {
-      @Override
-      public Void call() throws Exception {
-        if (options == null) {
-          mFileSystem
-              .createDirectory(new AlluxioURI(path), CreateDirectoryPOptions.getDefaultInstance());
-        } else {
-          mFileSystem.createDirectory(new AlluxioURI(path), options);
-        }
-        return null;
+    return RestUtils.call((RestUtils.RestCallable<Void>) () -> {
+      if (options == null) {
+        mFileSystem.createDirectory(new AlluxioURI(path),
+                CreateDirectoryPOptions.getDefaultInstance());
+      } else {
+        mFileSystem.createDirectory(new AlluxioURI(path), options);
       }
-    }, ServerConfiguration.global());
+      return null;
+    } , ServerConfiguration.global(), null);
```

Comment:
nit: can you remove the unnecessary space in "} ,"?

Reply:
done